### DISCUSSION TITLE
Option to make separate SVGs for (featured) repositories?

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#1381] [Ideas]
The repositories plugin mentions that due to limitations, svgs cannot link to individual repositories.

Perhaps, it could be an idea to have an option to basically separate each repository as its own SVG to wrap into a `<a>` tag?

The obvious downside would be that the beackend would need to do a lot more in terms of processing as the there would be more SVGs to generate.

It's just an idea and I understand if the drawbacks outweight the benefits and it wouldn't be realized.

**Comment by lowlighter** [comment]
Hi 👋 !

Yeah I think it's more practical for the user to just create multiple jobs for different SVGs output if they want to create linkable SVGs in their markdown.

An even better way would be to use the [markdown template](https://github.com/lowlighter/metrics/blob/master/source/templates/markdown/README.md) instead.
It'd be possible to generate multiple svg within the same run:
```ejs
<%- await embed(`repositories`, {repositories:true, plugin_repositories_starred:1}) %>
```

Some plugins also have a [markdown version](https://github.com/lowlighter/metrics/blob/master/source/templates/markdown/README.md#-plugins-with-markdown-version), this plugin could be a good candidate for this plugin
